src/composables/graph/useCanvasInteractions.ts

```diff
@@ -37,6 +39,35 @@ export function useCanvasInteractions() {
     // Otherwise, let the component handle it normally
   }
 
+  /**
+   * Handles pointer events from media elements that should potentially
+   * be forwarded to canvas (e.g., space+drag for panning)
+   */
+  const handlePointer = (event: PointerEvent) => {
+    // Check if canvas exists using established pattern
+    const canvas = getCanvas()
+    if (!canvas) return
+
+    // Forward space+drag (panning) events to canvas
+    // The canvas already handles space key state via read_only property
+    if (canvas.read_only && event.buttons === 1) {
+      event.preventDefault()
+      event.stopPropagation()
+      forwardEventToCanvas(event)
+      return
+    }
+
+    // Forward middle mouse button events for panning
+    if (event.buttons === 4) {
+      event.preventDefault()
+      event.stopPropagation()
+      forwardEventToCanvas(event)
+      return
+    }
```

Comment:
Nit: How do you feel about this, to make it clear that there are two conditions that both result in forwarding the event but that there's no difference otherwise?
```suggestion
    if (
      (canvas.read_only && event.buttons === 1) // Space + LMB 
      || (event.buttons === 4) // Forward middle mouse button events for panning
    ) {
      event.preventDefault()
      event.stopPropagation()
      forwardEventToCanvas(event)
      return
    }
```

Reply:
Done in https://github.com/Comfy-Org/ComfyUI_frontend/pull/5279/commits/d8f6021b3087914e0c1328829804a52f833e422f!